filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -268,6 +268,7 @@ func TestAuthorizeRequestFilter(t *testing.T) {
 				}
 				assert.Equal(t, ti.body, string(body))
 			}))
+			defer clientServer.Close()
```

Comment:
Is this the only place that is missing test server shutdown?

Reply:
I haven‘t checked yet. First I want to make sure this even fixes one test…